.env.example

```diff
@@ -1,6 +1,7 @@
 CLIENT_SECRET=The secret for sending to our OAuth server for logging in
 REFILE_REQUEST_SECRET=The secret for requesting NYPL platform APIs. Please be aware of the environment of the platform
-PLATFORM_BASE_URL=The base URL of the APIs of NYPL's microservices
+PLATFORM_BASE_URL=The base URL of the APIs of NYPL's microservices on production platform
+DEV_PLATFORM_BASE_URL=The base URL of the APIs of NYPL's microservices on development platform
```

Comment:
Don't quite get why this isn't the same variable with different values depending on the environment. I'm a bit confused about the `NODE_ENV` variable later on. Might become clearer in future code/PRs.

Reply:
It has the logic to determine which URL to use based on the environment in` appConfig.js` line 25.

But I think since, we need both of these URLs to exist, we have to have them both in our .env?

`NODE_ENV` is for setting the environment when we run the app. If we run it without building the dist file, `NODE_ENV` is `development`. Otherwise it is `production`. The reason is to better simulating how the app will be built on AWS.

On the other hand, `APP_ENV` is to determine which version of API endpoint we are going to call. So far, as the endpoint of refile is only on dev, so `APP_ENV` is set as `development`